network/src/protocols/discovery/test.rs

```diff
@@ -29,10 +29,28 @@ fn gen_full_node_payload() -> FullNodePayload {
     payload
 }
 
+fn get_addrs_from_note(note: &Note) -> Vec<Multiaddr> {
+    let signed_peer_info = note.get_signed_peer_info();
+    let peer_info: PeerInfo = protobuf::parse_from_bytes(signed_peer_info.get_peer_info()).unwrap();
```

Comment:
unwrap in a test is probably okay, but prefer expect so you get a sensible error message.

Reply:
Hmm, I'm not sure how I feel about this. My knee jerk reaction is that this seems noisy and low-value, since this is neither test related nor (realistically) error-able (assuming our protobuf library can read its own serialized output).

I'm probably wrong though :)

/cc @bothra90 ?

Reply:
Yeah, I don't see much value from adding an expect as compared to an unwrap here